### PR TITLE
Update icons

### PR DESCRIPTION
**Pull request by araiford**
This PR adds a couple icons being used in navigation prototypes, including:
- shield-lock
- heartbeat-outline
- heartbeat-fill

This PR also deprecates the `monitoring` icon and renames it to `chart`.

### LGTM's
* [ ] Dev LGTM
* [ ] Dev LGTM

### CODE REVIEW

File: docs/_data/icons.json5 [hunk 1]
```diff
@@ -59,8 +59,12 @@
         ],
     },
     {
-        name: 'checkmark',
-        minver: '0.1.0',
+      name: 'chart',
+      minver: '0.19.0',
```
Comment (100stacks) [inline docs/_data/icons.json5:63]: Please change to our next release:

```json
minver: '0.20.0',
```

File: docs/_data/icons.json5 [hunk 2]
```diff
@@ -148,11 +152,25 @@
...
-        ],
+      name: 'help-circle',
+      minver: '0.1.10',
+      keywords: [
+        'question',
+      ],
+    },
+    {
+      name: 'heartbeat-fill',
+      minver: '0.19.0',
```
Comment (100stacks) [inline docs/_data/icons.json5:163]: Please change to our next release:

```json
minver: '0.20.0',
```

File: docs/_data/icons.json5 [hunk 3]
```diff
@@ -148,11 +152,25 @@
...
+      keywords: [
+        'monitoring',
+      ],
+    },
+    {
+      name: 'heartbeat-outline',
+      minver: '0.19.0',
```
Comment (100stacks) [inline docs/_data/icons.json5:170]: Please change to our next release:

```json
minver: '0.20.0',
```

File: docs/_data/icons.json5 [hunk 4]
```diff
@@ -478,6 +498,13 @@
         name: 'server',
         minver: '0.1.10',
     },
+    {
+      name: 'shield-lock',
+      minver: '0.19.0',
```
Comment (100stacks) [inline docs/_data/icons.json5:503]: Please change to our next release:

```json
minver: '0.20.0',
```

File: docs/_data/icons.json5 (hunk 1 above)
Comment (araiford) [reply docs/_data/icons.json5]: Updated

**Pull request merged**